operator/src/main/resources/application.properties

```
@@ -4,7 +4,7 @@ quarkus.docker.dockerfile-jvm-path=Dockerfile
 quarkus.operator-sdk.crd.validate=false
 
 # Operator config
-operator.keycloak.image=quay.io/keycloak/keycloak:nightly
+operator.keycloak.image=${RELATED_IMAGE_KEYCLOAK:quay.io/keycloak/keycloak:nightly}
```

```suggestion
operator.keycloak.image=${RELATED_KEYCLOAK_IMAGE:quay.io/keycloak/keycloak:nightly}
```
Nitpick: Would be good to keep some sort of convention to the naming. WDYT?

---

RELATED_IMAGE_something is the required convention here.

---

@shawkins Ah, OK, thanks for the explanation. Ignore the suggestion.